hack/publish_package_builds.sh

```diff
@@ -106,14 +125,24 @@ TARGET_BUNDLE_IMAGE="${BUNDLE_BASE}:${TARGET_TAG}"
 TARGET_CATALOG_IMAGE="${CATALOG_BASE}:${TARGET_TAG}"
 
 CSV_FILE="bundle/manifests/lightspeed-operator.clusterserviceversion.yaml"
-CATALOG_FILE="lightspeed-catalog/operator.yaml"
+CATALOG_FILE="lightspeed-catalog/index.yaml"
 CATALOG_DOCKER_FILE="lightspeed-catalog.Dockerfile"
+CATALOG_INTIAL_FILE="hack/operator.yaml"
+KUSTOMIZATION_FILE="config/manager/kustomization.yaml"
+OLS_CONFIG_FILE="bundle/manifests/ols.openshift.io_olsconfigs.yaml"
 
 echo "Backup files before running the script"
-backup
-trap restore EXIT
+
+backup "${REGENERATE}"
+trap 'restore "${REGENERATE}"' EXIT
 
 
+if [[ ${REGENERATE} == "true" ]]; then
+  make docker-build docker-push VERSION="${VERSION}" IMG="${OPERATOR_IMAGE}"
+  rm -rf ./bundle
+  make bundle VERSION="${VERSION}" BUNDLE_IMG="${TARGET_BUNDLE_IMAGE}"
```

Comment:
make bundle gets called regardless a few lines further down, why is it needed here?

Reply:
We dont need it  here . Removed .